### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by sago35**
Update!

### CODE REVIEW

**Comment by sago35** [comment]
Now ready for review

**Comment by doniacld** [comment]
LGTM! Nicer README 🎉

File: cmd/tinyfontgen-ttf/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+## Examples
+
+### RubikMoonrocks
+
+https://fonts.google.com/specimen/Rubik+Moonrocks?query=rub
+
+```
+$ go run ./cmd/tinyfontgen-ttf --size 32 --package rubikmoonrocks --fontname RubikMoonrocks ./RubikMoonrocks-Regular.ttf --output rubikmoonrocks/font.go
```
Comment (doniacld) [inline cmd/tinyfontgen-ttf/README.md:52]: We could generate an executable when releasing.

**Comment by conejoninja** [comment]
Thanks for your contribution @sago35 , merged!

**Pull request merged**